jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -73,6 +74,53 @@ local excludedRules = [
       { alert: 'PrometheusErrorSendingAlertsToAnyAlertmanager' },
     ],
   },
+  {
+    name: 'kube-prometheus-node-recording.rules',
+    rules: [
+      { alert: 'instance:node_cpu:rate:sum' },
+      { alert: 'instance:node_network_receive_bytes:rate:sum' },
+      { alert: 'instance:node_cpu:ratio' },
+      { alert: 'instance:node_network_transmit_bytes:rate:sum' },
```

Comment:
same remark about the OCP console using these rules (except `instance:node_cpu:ratio` apparently).